index.html

```diff
@@ -73,7 +74,231 @@ <h3>LET'S CONNECT</h3>
+                </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- End: Work Section-->
+
+        <!-- Start: About Section-->
+
+        <section id="about-section" class="about">
+            <div class="about-left">
+                <h2>About Myself</h2>
+                <p>
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website. Look through some of my work and experience! If
+                you like what you see and have a project you need coded, don’t
+                hesitate to contact me.
+                </p>
+                <h3>LET’S CONNECT</h3>
+                <ul class="social-media">
+                <li>
+                    <a href="#"
+                    ><img src="images/twitter-icon.svg" alt="Twitter icon"
+                    /></a>
+                </li>
+                <li>
+                    <a href="#"
+                    ><img src="images/linkedin-icon.svg" alt="Linkedin icon"
+                    /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/m-icon.svg" alt="M icon" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/peace-icon.svg" alt="Peace icon" /></a>
+                </li>
+                <li>
+                    <a href="#"
+                    ><img src="images/github-icon.svg" alt="Github icon"
+                    /></a>
+                </li>
+                </ul>
+                <div class="get-resume-btn">
+                <input type="button" value="Get my resume" />
+                </div>
+            </div>
+
+            <div class="about-right">
+                <ul>
+                <li class="skills">
+                    <p><strong>Languages</strong></p>
+                    <i class="fa fa-chevron-down"></i>
+                </li>
+                <li class="skills skills-inner">
+                    <ul>
+                    <li>
+                        <img src="images/js.svg" />
+                        <p>JavaScript</p>
+                    </li>
+                    <li>
+                        <img src="images/html.svg" />
+                        <p>HTML</p>
+                    </li>
+                    <li>
+                        <img src="images/css.svg" />
+                        <p>CSS</p>
+                    </li>
+                    </ul>
+                </li>
+                <li class="skills">
+                    <p><strong>Frameworks</strong></p>
+                    <i class="fa fa-chevron-right"></i>
+                </li>
+                <hr />
+                <li class="skills">
+                    <p><strong>Skills</strong></p>
+                    <i class="fa fa-chevron-right"></i>
+                </li>
+                <hr />
+                </ul>
+            </div>
+            </section>
```

Comment:
- [ ] As mentioned in the project requirement, you should implement everything as seen in the Figma design. Notice the missing accordion in your design and the gray color horizontal line after the `skill`. Kindly fix this 

Your design | Figma Design
----- | -----
<img width="330" alt="image" src="https://github.com/gfrancoarq/portfolio2023/assets/84156714/19f99f10-342e-4d1d-b735-30832aaf2b53"> | <img width="266" alt="image" src="https://github.com/gfrancoarq/portfolio2023/assets/84156714/cf2857e2-621f-4775-a6d4-68ba4175ba8a">